gce/setup-guide/secret_config.yaml

```diff
@@ -11,9 +11,11 @@ jupyterhub:
       clientId: "<CLIENTID>"
       clientSecret: "<CLIENT_SECRET>"
       callbackUrl: "http://<IP_OR_DOMAIN>/hub/oauth_callback"
-      org_whitelist:
-        - your_github_org
-        - pangeo-data
+        orgWhitelist:
```

Comment:
Hmm. This seems wrong. It looks like setting it this way (based on the z2jh guide) still gives access to any Github user.

Reply:
Here: https://zero-to-jupyterhub.readthedocs.io/en/latest/authentication.html#giving-access-to-organizations-on-github

Reply:
cc @raphaeldussin and @rabernat who have recently been trying to sort out this problem.

Reply:
See also: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1118

Reply:
OK. Seems like it does seem to be `org_whitelist` for now, but will eventually change to `orgWhitelist` when this catches up with z2jh.